platforms-serverless-vercel/vercel-with-redwood/api/jsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
     "noEmit": true,
-    "allowJs": true,
     "esModuleInterop": true,
     "target": "esnext",
     "module": "esnext",
     "moduleResolution": "node",
+    "skipLibCheck": false,
```

Comment:
Why is it needed for Redwood?

Reply:
It was added by Redwood, not sure. Maybe for performance?

Reply:
I see, and less problems from users too maybe 😄